GoodDollar/GoodDAPP
# (Feature) User should see a message about face verification disabled for 24 hours

# Description

[x] in the "verify me" screen call server endpoint
[x] disable button until have response
[x] if account is being deleted show popup. on dismiss go back to dashboard.
[x] if account not being deleted enable button

About #1984 

# How Has This Been Tested?

1. signup
2. try to claim
3. on the FR intro screen "ok verify" button should be disabled for a while then enables
4. pass FR & claim
5. delete wallet
6. signup again
7. try to claim
8. on the FR intro screen "ok verify" button should be disabled for a while then popup appears
9. once popup dismissed we should be redirected back to the dashboard

# Checklist:


## Screenshot:
<img width="488" alt="1" src="https://user-images.githubusercontent.com/49862004/85373142-8490c700-b53b-11ea-8f07-ac0324914f6a.png">
